lib/shared/src/chat/context-filter.ts

```diff
@@ -0,0 +1,64 @@
+export const CodyIgnoreFileName = '.codyignore'
```

Comment:
```suggestion
export const CODY_IGNORE_FILENAME = '.codyignore'
```

We usually name constants in uppercase to visually distinguish it from classes.